README.md

```diff
@@ -157,16 +158,16 @@ Check out our [**Contributing Guide**](https://www.gatsbyjs.com/contributing/how
 
 ### A note on how this repository is organized
 
-This repository is a [monorepo](https://trunkbaseddevelopment.com/monorepos/) managed using [Lerna](https://github.com/lerna/lerna). This means there are [multiple packages](/packages) managed in this codebase, even though we publish them to NPM as separate packages.
+This repository is a [monorepo](https://trunkbaseddevelopment.com/monorepos/) managed using [Lerna](https://github.com/lerna/lerna). This means there are [multiple packages](https://github.com/gatsbyjs/gatsby/tree/master/packages) managed in this codebase, even though we publish them to NPM as separate packages.
 
-### Contributing to Gatsby v1
+### Contributing to Gatsby v2
 
-We are currently only accepting bug fixes for Gatsby v1. No new features will be accepted.
+We are currently only accepting bug fixes for Gatsby v2. No new features will be accepted.
 
-## :memo: License
+## 📝 License
 
 Licensed under the [MIT License](./LICENSE).
 
 ## 💜 Thanks
 
-Thanks to our many contributors and to [Netlify](https://www.netlify.com/) for hosting [Gatsby](https://www.gatsbyjs.com) and our example sites.
+Thanks to our many contributors who create plugins, starters, videos, blog posts, and anything else you can think of. We also really appreciate any community members who help with issues & PRs inside our repository or answer questions on Discord & GitHub Discussions. A big part of what makes Gatsby great is each and every one of you in the community. Your contributions enrich the Gatsby experience and make it better every day.
```

Comment:
", and anything else you can think of" is too informal. ", etc." would be great. The rest is perfect.